scripts/lightweight_curator_test.py

```diff
@@ -1,21 +1,115 @@
-from curator import env_validation, log
-import pytest
-
-class TestLog:
-    def test_log_fail(self):
-        log("fake level", "Dummy log message") == None
-
-    def test_log_pass(self, capsys):
-        log("info", "Dummy log message") == {"level": "info", "message": "Dummy log message"}
-        captured = capsys.readouterr()  # Capture output
-        assert captured.out == '{"level": "info", "message": "Dummy log message"}\n'
-
-class TestEnvValidation:
-    def test_env_validation_fail(self):
-        with pytest.raises(SystemExit) as pytest_wrapped_e:
-                env_validation(2, "", "elasticsearch:9200")
-        assert pytest_wrapped_e.type == SystemExit
-        assert pytest_wrapped_e.value.code == 1
-
-    def test_env_validation_pass(self):
-        assert env_validation(2, "infra-,app-,audit-", "elasticsearch:9200") == None
+# -*- coding: utf-8 -*-
+
+import elasticsearch
+import unittest
+from datetime import datetime
+from elasticmock_additional_apis import elasticmock, behaviour
+
+INDEX_NAME = 'test_index'
+DOC_TYPE = 'doc-Type'
+BODY = {
+    'author': 'tgabriel',
+    'text': 'Elasticsearch: cool. bonsai cool.',
+    'timestamp': datetime.now(),
+}
+
+class TestElasticmock(unittest.TestCase):
+
+    @elasticmock
+    def setUp(self):
+        self.es = elasticsearch.Elasticsearch(hosts=[{'host': 'localhost', 'port': 9200}])
```

Comment:
why not add the the behaviour here instead of doing it only once?

not sure if it works though